docs/code_samples/helloworld_entity_world.py

```diff
@@ -0,0 +1,8 @@
+from . import _main_code_sample
+
+# Output file name must be unique!
+output_file_name = "Entity_World.php"
+# Ensure that the URL always starts with https://raw.githubusercontent.com/...
+url = "https://raw.githubusercontent.com/mautic/plugin-helloworld/mautic-4/Entity/World.php" 
```

Comment:
Do we need similar py file for each code sample?

Reply:
Yes, just like all external links have their own files in the `docs/links` folder. We could also add them all into a single file, but that file might become quite large over time. I don't have a strong preference.

Reply:
I was just checking because of the other comment:

https://github.com/mautic/developer-documentation-new/pull/88/files#r1038801518

So it needs that small block + the py file to create a new embed-able sample